app/src/main/java/io/github/droidkaigi/confsched2017/view/helper/DataBindingHelper.java

```diff
@@ -83,6 +85,18 @@ public static void setTextLinkify(TextView textView, boolean isLinkify) {
         }
     }
 
+    @BindingAdapter("url")
```

Comment:
`url` is a little ambiguous.
BindingAdapter has no namespace, so how about adding prefix like "webViewUrl"?